common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -52,7 +52,7 @@ case class UsageRightsToMetadataParser(resources: ImageProcessorResources) exten
 class PhotographerParser(photographersConfig: UsageRightsConfigProvider) extends ImageProcessor {
   def apply(image: Image): Image = {
     image.metadata.byline.flatMap { byline =>
-      photographersConfig.getPhotographer(byline).map{
```

Comment:
presumably in theory the fallback to `image.uploadTime` could be a lot later than the actual date the photo was taken (e.g. if someone was uploading archival photos)

i'm guessing this has been discussed with relevant people already, though?

Reply:
Yep, while imperfect I think this is fine as new images are more likely to be uploaded than archives, so better to apply the wrong metadata and let uploader find and correct it, than to not apply it all to the images where it would be correct. It's also no more wrong than current behaviour which would do the same.